modules/ROOT/pages/ip-blacklist.adoc

```diff
@@ -53,11 +61,32 @@ You can use a Mule expression to instruct the service to look for the IP address
 #[message.inboundProperties['http.headers']['X-Forwarded-For']]
 ----
 
-Insert the IP address in the IP expression field:
+When you insert the IP address in the IP expression field of the policy parameters, the API Platform is instructed to look at the first IP address that ranks in the concatenated values of the `x-Forwarded-For` header of the request:
 
 image::ip-whitelist-ip-expression.png[ip+blacklist+ip_expression]
 
-This instructs API Platform to look at the first IP address that ranks in the concatenated values of the 'x-Forwarded-For' header of the request.
+
+== Configuring Policy Parameters
+
+When you apply the IP Blacklist policy to your API from the UI, you can configure the following parameters:
+
+[%header%autowidth.spread,cols="a,a,a"]
+|===
+| Parameter | Description 
+|IP expression  
+|The Mule expression to be used for extracting the IP address from this API request. 
+|Blacklist
+|The list of IP addresses that must be denies access to the API. Add one or several IP addresses separated by a comma, or define a range. For more information, see <<specifying_ip_address,Specifying IP Addresses to Blacklist in the Policy>>.
```

Comment:
Separated by comma? The UI does not allow it. You may define one IP or IP range at a time, as many times you need it, while configuring the policy .

Reply:
Done! Thanks!